core/main/src/Core/Particle/Mover.ts

```diff
@@ -36,6 +37,12 @@ export class Mover {
         this.moveParallax();
     }
 
+    public moveXY(x: number, y: number): void {
```

Comment:
You can use this method also in lines 80/81

Reply:
Done